src/revisions_parser.py

```diff
@@ -0,0 +1,123 @@
+import re
+from typing import Dict, List
+
+
+def month_to_number(date: str) -> str:
```

Comment:
There is not a check for None in date variable. There is however e.g. clean_date() function with the same parameter and it has a None check. It would be most likely better, if this was consistent.